web/styles/compose.css

```diff
@@ -263,14 +263,51 @@
     }
 }
 
-#compose-textarea,
-#preview-message-area-container {
-    grid-area: compose-textarea;
-    /* When the compose box is full screen, this
-       auto height will kick in and make sure that
-       the textarea opens to fill its whole grid
-       area. */
-    height: auto;
+#message-content-container {
+    grid-area: message-content-container;
+    display: grid;
+    grid-template: minmax(0, 1fr) / minmax(0, 1fr) 25px;
+    margin-top: 5px;
```

Comment:
I'd love to see this `margin-top` go away in some future refactor that uses grid in place of flex on `.compose_table`. But for the time being, we can actually get a better vertically-aligned Drafts button if that `margin-top` is moved over to `.messagebox`:

![Screenshot 2024-04-24 at 2 34 27 PM](https://github.com/zulip/zulip/assets/170719/f9ea9e8e-ddab-49c8-9cd0-d8a519172721)